### ISSUE TITLE
Wonder list with mods - not split in eras

### ISSUE THREAD

**Issue by krijnsent** [yairm210/Unciv#8693]
**Platform**
Windows, 64 bits

**Version**
4.4.15

**Describe the bug**
List of wonders (Overview -> Wonders) is built up per era for the normal Civ 5 game, but for mods there seems to be no ordering. See e.g. the Civ 6 mod or the DeCiv Redux mod, in both cases the wonders are one massive list (basically showing all wonders at the start of the game, not just the ones of the first era, like when playing normal Civ5), not split into eras. It could be the mods, but the techs.json & buildings.json seem okay.

**Comment by RealBamboolord** [comment]
Not a bug, since those wonders have no tech requirement

**Comment by yairm210** [comment]
This sounds like a mod issue, there's no problem with Unciv

**Comment by krijnsent** [comment]
@yairm210 I dove a bit deeper, it looks like the wonders of mods show in a complete list until the player arrives in an era, at that point the "sub-section" for that era appears. That means that when you start with a mod, you have one big list of all the wonders, all showing as "ancient era".

See this screenshot: I'm in the Classical era, but when I keep on playing...
![image](https://user-images.githubusercontent.com/21982451/219938623-076e9a6a-fb1b-45a4-bc12-3afb28c43abc.png)

This is the next overview (different game, much later in the game, same mod, civ6):
![image](https://user-images.githubusercontent.com/21982451/219938675-441b252f-9e2f-482e-8d3b-5b9427336ac8.png)

In a nutshell: 
- in the Civ5, only the wonders of the era a player is in show, later wonders don't show
- with mods, all wonders show from the start as if they all are in the ancient era, but when the player progresses in the tech tree, the wonders of the eras you have reached are subdivided into those eras.

That difference is rather confusing and is a bug in my eyes - I like the Civ5 setup where you only see the wonders of the eras you've reached, which means filtering those out for mods.

**Comment by yairm210** [comment]
Hm
Can you send me an example save?

**Comment by krijnsent** [comment]
Using the Civ6 mod -
https://www.dropbox.com/s/93yule2kpthwt8d/Mali%20-%2088%20turns?dl=0 - Mali, turn 88
https://www.dropbox.com/s/4on9nn9w2o3kf7f/Australia%20-%20318%20turns?dl=0 - Australia, turn 318

Open them one after the other and note the difference in the wonder-list, mainly the missing era-grouping.

**Comment by yairm210** [comment]
This is indeed a mod problem.
These wonders have no tech defined, therefore we put them in the latest era.

We can make this clearer visually by putting them in a different category but if you want them in an era you need to give them a tech.

**Comment by yairm210** [comment]
Or just put them in the first era so they remain consistently there, I think that's an ok compromise

**Comment by yairm210** [comment]
I see that those wonders require specific policies, so it's probably not relevant to put them under the ancient era - maybe "other"...?

**Comment by krijnsent** [comment]
Yeah, what would make most sense to me:
- put the ones with an era (e.g. for the Civ6 mod there are plenty of those) in the appropriate era
- for all non-era related wonders (other requirements), add a rest category at the bottom -  "other"
